internal/docker/docker.go

```diff
@@ -14,7 +15,7 @@ func RunDockerCommand(workingDir string, showCommand bool, pipeStdout bool, comm
 }
 
 func RunDockerComposeCommand(workingDir string, showCommand bool, pipeStdout bool, command ...string) error {
-	dockerCmd := exec.Command("docker-compose", command...)
+	dockerCmd := exec.Command("docker", append([]string{"compose"}, command...)...)
```

Comment:
```suggestion
	dockerCmd := exec.Command("docker-compose", command...)...)
```
Unfortunately docker in GitHub Actions does not support `docker compose` yet. I'm not sure what version of Docker they are running there, but it appears to be older.